packages/utils-expression-evaluator/lib/expressions/Term.ts

```diff
@@ -428,13 +422,9 @@ export class NonLexicalLiteral extends Literal<{ toString: () => 'undefined' }>
   }
 
   public override coerceEBV(): boolean {
-    const isNumericOrBool =
-      isSubTypeOf(this.dataType, TypeURL.XSD_BOOLEAN, this.openWorldType) ||
-      isSubTypeOf(this.dataType, TypeAlias.SPARQL_NUMERIC, this.openWorldType);
-    if (isNumericOrBool) {
-      return false;
-    }
-    throw new Err.EBVCoercionError(this);
+    // Always throws in [SPARQL 1.2](https://www.w3.org/TR/sparql12-query/#ebv),
```

Comment:
I'm guessing this can still change, based on the resolution of https://github.com/w3c/sparql-query/issues/286

Reply:
Yes, it can, but if they decide to change this, the spec tests will also change. So I would NOT wait for this PR until that one is closed.